apps/payments/next/app/[locale]/[offeringId]/[interval]/upgrade/[cartId]/(startLayout)/start/page.tsx

```diff
@@ -97,6 +97,13 @@ export default async function Upgrade({
                 width={91}
                 height={24}
               />
+            ) : cart.paymentInfo.type === 'link' ? (
+              <Image
+                src={getCardIcon('link', l10n).img}
+                alt={l10n.getString('link-logo-alt-text', 'Link logo')}
+                width={70}
+                height={20}
```

Comment:
suggestion (non-blocking): Seems like there's an attempt to standardize around height=24. Could the change be made here as well, please?